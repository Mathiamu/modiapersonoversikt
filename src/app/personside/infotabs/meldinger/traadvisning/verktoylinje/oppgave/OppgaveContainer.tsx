import * as React from 'react';
import RestResourceConsumer from '../../../../../../../rest/consumer/RestResourceConsumer';
import { GsakTema } from '../../../../../../../models/meldinger/oppgave';
import OppgavePanel from './OppgavePanelContainer';

function OppgaveContainer() {
    return (
        <RestResourceConsumer<GsakTema[]> getResource={restResources => restResources.oppgaveGsakTema}>
            {data => <OppgavePanel gsakTema={data} />}
        </RestResourceConsumer>
    );
}

export default OppgaveContainer;
