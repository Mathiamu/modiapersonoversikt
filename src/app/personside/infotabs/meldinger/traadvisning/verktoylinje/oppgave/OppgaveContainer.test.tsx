import * as React from 'react';
import * as renderer from 'react-test-renderer';
import TestProvider from '../../../../../../../test/Testprovider';
import OppgaveContainer from './OppgaveContainer';

test('Viser oppgavecontainer med alt innhold', () => {
    const container = renderer.create(
        <TestProvider>
            <OppgaveContainer lukkPanel={() => {}} />
        </TestProvider>
    );

    const json = container.toJSON();
    expect(json).toMatchSnapshot();
});
