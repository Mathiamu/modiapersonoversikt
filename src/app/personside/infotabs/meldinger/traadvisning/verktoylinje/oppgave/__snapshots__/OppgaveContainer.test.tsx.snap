// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viser oppgavecontainer med alt innhold 1`] = `
.c0 {
  margin-top: 1.5rem;
}

<div
  className="c0"
/>
`;
